test/lib/build/definitions/library.js

```diff
@@ -606,3 +630,89 @@ test("buildThemes: CSS Variables enabled", (t) => {
 	t.is(taskUtil.getBuildOption.getCall(0).args[0], "cssVariables",
 		"taskUtil#getBuildOption got called with correct argument");
 });
+
+test("Standard build: nulled taskFunction to skip tasks", (t) => {
+	const {project, taskUtil, getTask} = t.context;
```

Comment:
should we have a test to check assigning a custom task to any possible task?

Reply:
Thanks! I have updated the TasksRunner's tests with that.